### PR TITLE
Updates for policy sample removal in doc, adding READMEs

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/8269

### CODE REVIEW

File: stable/CM-Configuration-Management/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Configuration management policies
+
+Use configuration management policies maintain a consistent state for your cluster. View the following table list of the stable policies that are supported by [Red Hat Advanced Cluster Management for Kubernetes](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.1/html-single/security/index#kubernetes-configuration-policy-controller):
```
Comment (dockerymick) [inline stable/CM-Configuration-Management/README.md]: Change link to GRC page

File: stable/CA-Security-Assessment-and-Authorization/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Security Assessment and Authorization policies
+
+See [NIST Special Publication 800-53 (Rev. 4)](https://nvd.nist.gov/800-53/Rev4/control/CA-1) for a description of the Security Assessment and Authorization security control.
```
Comment (dockerymick) [inline stable/CA-Security-Assessment-and-Authorization/README.md]: Belongs the CA- https://nvd.nist.gov/800-53/Rev4/family/Security%20Assessment%20and%20Authorization

update the link

**Review by ycao56 — APPROVED** [review]

File: stable/CM-Configuration-Management/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Configuration Management 
+
+See [NIST Special Publication 800-53 (Rev. 4)](https://nvd.nist.gov/800-53/Rev4/family/Configuration%20Management) for a description of the Configuration Management control. View the configuration policies that map to the Configuration Management catalog, and helps to detect or remediate security issues that are in the category. See the following table list of the stable policies that are supported by [Red Hat Advanced Cluster Management for Kubernetes](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.1/html/security/security#governance-and-risk):
```
Comment (dockerymick) [inline stable/CM-Configuration-Management/README.md:3]: @ycao56 Can you confirm that we are referencing the correct link. Is there a specific control number that we want to link to? For example, [CM-9](https://nvd.nist.gov/800-53/Rev4/control/CM-9).

Comment (ycao56) [reply stable/CM-Configuration-Management/README.md:3]: yes, https://nvd.nist.gov/800-53/Rev4/family/Configuration%20Management is the right link. CM-9 is sub catogery

**Pull request merged**